README.txt

```diff
@@ -19,7 +19,7 @@ To ensure the quality of the plugin please let me know if you encounter any issu
 ### Features
 
 * The plugin calculates the dominant color of an image upon upload.
-* All images attached to posts and pages are automatically replaced with placeholders and load as soon as they enter the viewport to save bandwidth.
+* All images attached to posts and pages are automatically replaced with placeholders and load as soon as they enter the viewport to save bandwidth. To disable placeholders for certain images you can use the class disable-dcll.
```

Comment:
I usually copy the README.md exactly as it is to the .txt for the WordPress page, no need to change formatting 😄